clients/venice-push-job/src/main/java/com/linkedin/venice/spark/SparkConstants.java

```diff
@@ -20,6 +20,16 @@ public class SparkConstants {
       new StructField[] { new StructField(KEY_COLUMN_NAME, BinaryType, false, Metadata.empty()),
           new StructField(VALUE_COLUMN_NAME, BinaryType, true, Metadata.empty()) });
 
+  public static final StructType KAFKA_INPUT_TABLE_SCHEMA = new StructType(
```

Comment:
Would we need to version this at all?

Reply:
My plan is to keep this as is for the most part ( after getting a few runs in EI ) . and for the consistency checker I'll have an extended form of the table schema that contains the offset vectors and all the goodies prebaked and present for consistency checker. 
If there are any specifics that you think we need for this ( and the ETL functionality ) Please advise.

Reply:
I guess what I'm wondering, do we need to be concerned if this struct type changes....  my initial thinking is that it's probably ok since this used to generate NEW datasets, not process old ones?